src/java/org/apache/cassandra/audit/AuditLogOptions.java

```diff
@@ -60,6 +300,7 @@ public String toString()
                ", block=" + block +
                ", max_queue_weight=" + max_queue_weight +
                ", max_log_size=" + max_log_size +
+               ", max_archive_retries=" + max_archive_retries +
```

Comment:
Not sure whether this change might not break someone's parsing actually?

Reply:
hardly, nobody parses this anyway ... I am fixing things here acutally :)